detox/test/integration/utils/simplistic-runner.js

```diff
@@ -11,11 +12,16 @@ async function main() {
 }
 
 async function test1() {
+  log.info('Test 1');
   await device.launchApp();
 }
 
 async function test2() {
+  log.info('Test 2');
   await device.terminateApp();
 }
 
-main().catch((e) => console.error(e));
+main().catch((e) => {
+  console.error(e);
```

Comment:
Without the fix to the `BunyanLogger` class, the `console.error` just does not print anything. This is why I got interested and found the reason and fixed it.